README.md

```diff
@@ -36,10 +36,14 @@ As mentioned in the command output, don't forget to add the installation directo
 
     2. manually from git, or download from https://www.rust-lang.org/install.html (the 'rustc' source download behind the 'source' link is the right one).
```

Comment:
Now we can't install source from `https://www.rust-lang.org/install.html`, so could you please modify this sentence to just 'manually from [git](https://github.com/rust-lang/rust.git)' ?

Reply:
ack